low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml

```diff
@@ -13,48 +13,38 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-[SYSTEM_INFORMATION]
-rack_id=001
-site_id=001
-cluster_id=001
-node_id=001
-product=LDR_R2
-
-[SSPL-TESTS_SETTING]
-modules=RabbitMQingressProcessorTests, RabbitMQegressProcessor
-
-[RABBITMQEGRESSPROCESSOR]
-virtual_host=SSPL
-queue_name=actuator-req-queue
-exchange_name=sspl-in
-routing_key=actuator-req-key
-username=sspluser
-password=sspl4ever
-message_signature_username=sspl-ll
-message_signature_token=ALOIUD986798df69a8koDISLKJ282983
-message_signature_expires=3600
-primary_rabbitmq_host=localhost
-
-[RABBITMQINGRESSPROCESSORTESTS]
-virtual_host=SSPL
-queue_name=sensor-queue
-exchange_name=sspl-out
-routing_key=sensor-key
-username=sspluser
-password=sspl4ever
-primary_rabbitmq_host=localhost
-
-[RAIDSENSOR]
-monitor=true
-
-[REALSTORSENSORS]
-monitor=true
-
-[NODEHWSENSOR]
-monitor=true
-
-[SYSTEMDWATCHDOG]
-monitor=true
-
-[RABBITMQCLUSTER]
-cluster_nodes=localhost
+cluster:
```

Comment:
Refer review comments from https://github.com/Seagate/cortx-monitor/pull/282, for all imported or duplicated files

Reply:
Done